lib/spack/spack/test/util/environment.py

```diff
@@ -54,6 +57,7 @@ def test_prune_duplicate_paths():
     assert(expected == envutil.prune_duplicate_paths(test_paths))
 
 
+@pytest.mark.skipif(sys.platform == 'win32', reason="All Fetchers Failed")
```

Comment:
The `reason` string seems strange to me for these tests: do they actually fail because of issues with fetching?

Reply:
This module is now passing.